src/network/networkd-ndisc.c

```diff
@@ -913,10 +913,6 @@ static int ndisc_router_process_reachable_time(Link *link, sd_ndisc_router *rt)
         if (!link->network->ndisc_use_reachable_time)
                 return 0;
 
-        /* Ignore the reachable time field of the RA header if the lifetime is zero. */
-        r = sd_ndisc_router_get_lifetime(rt, NULL);
-        if (r <= 0)
-                return r;
 
```

Comment:
Please drop the doubled empty line.

Reply:
Thanks for spotting that.  I've removed all double empty lines.